Nayara-Oliveira/exercicio-casa/sistema_de_alunas.py

```diff
@@ -0,0 +1,143 @@
+from dataset_alunas import dataset
+
+def main():
+    print("\n---  Seja bem vinda a Escola do Reprograma!  ---")
+    print("Sistema de informações de alunas")
+    
+    while True:
+        cod_opcao = obter_opcao()
+        
+        if cod_opcao == 1: incluir_nova_aluna()
+        elif cod_opcao == 2: consultar_lista_alunas()
+        elif cod_opcao == 3: consultar_faltas_aluna()
+        elif cod_opcao == 4: consultar_notas_aluna()
+        elif cod_opcao == 5: consultar_status_aprovacao()
+        elif cod_opcao == 6: print("Encerrando o programa..."); break
+
+def obter_opcao():
+    codigo_opcao = 0
+
+    while codigo_opcao not in [1, 2, 3, 4, 5]:
+        try:
+            codigo_opcao = int(input("\nEscolha uma opção:\n"
+                                    "1 - Incluir uma nova aluna\n"
+                                    "2 - Consultar lista de alunas\n"
+                                    "3 - Consultar faltas da aluna\n"
+                                    "4 - Consultar notas da aluna\n"
+                                    "5 - Consultar status de aprovação\n"
+                                    "6 - Sair do sistema\n"
+                                    "Opção: "))
+                
+            if codigo_opcao not in [1, 2, 3, 4, 5]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 5).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+            
+        return codigo_opcao
+    
+def incluir_nova_aluna():
+    print("Insira os seguintes dados para cadastrar uma nova aluna:")
+    nome = input("Nome da aluna: ")
+    sobrenome = input("Sobrenome da aluna: ")
```

Comment:
A parte de obter nome e sobrenome da aluna e construir uma tupla é usada nas outras funções também, neste caso, poderiamos fazer uma função que recebe esses dados e retorna uma chave.

**Sugestão:**
```suggestion
    chave_aluna = obter_nome_aluna()

def obter_nome_aluna()
    nome = input("Nome da aluna: ")
    sobrenome = input("Sobrenome da aluna: ")
    return (nome, sobrenome)
```